qiskit/dagcircuit/dagnode.py

```diff
@@ -49,6 +49,26 @@ def __str__(self):
         return str(id(self))
 
     @staticmethod
+    @deprecate_arg(
+        "bit_indices1",
+        deprecation_description="Not setting bit_indices1 in DAGNode.semantic_eq()",
+        additional_msg=(
+            "To ease the transition, bit_indices1 and bit_indices2 will for now be "
+            "pre-populated based on the values found in Bit.index and Bit.register."
+        ),
+        since="0.21.0",
+        predicate=lambda arg: arg is None,
+    )
+    @deprecate_arg(
+        "bit_indices2",
+        deprecation_description="Not setting bit_indices2 in DAGNode.semantic_eq()",
+        additional_msg=(
+            "To ease the transition, bit_indices1 and bit_indices2 will for now be "
+            "pre-populated based on the values found in Bit.index and Bit.register."
+        ),
+        since="0.21.0",
+        predicate=lambda arg: arg is None,
+    )
```

Comment:
similar comment to another PR about the phrasing of the fixed message being maybe more confusing than just leaving a custom message in place